apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -19,7 +19,7 @@ class SectionAssessments extends Component {
   static propTypes = {
     // provided by redux
     sectionId: PropTypes.number.isRequired,
-    assessments: PropTypes.array,
+    assessments: PropTypes.object,
```

Comment:
It looks like `props.assessments` is now unused in this component - should we remove it entirely?  Or is this a temporary state?

Reply:
This entire component is temporary - Nkiru and I will be hooking up tables based on this API tomorrow. At that time, we'll define a set of PropShapes that we can use throughout the assessments tab.